keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -158,7 +161,8 @@
           "Keys"
         ],
         "operationId": "UpdateKey",
-        "description": "The update key operation changes specified attributes of a stored key and can be applied to any key type and key version stored in Azure Key Vault. The cryptographic material of a key itself cannot be changed. In order to perform this operation, the key must already exist in the Key Vault. Authorization: requires the keys/update permission.",
+        "summary": "The update key operation changes specified attributes of a stored key and can be applied to any key type and key version stored in Azure Key Vault.",
+        "description": "The cryptographic material of a key itself cannot be changed. In order to perform this operation, the key must already exist in the Key Vault.",
```

Comment:
I probably should have worded it as this comment https://github.com/Azure/azure-rest-api-specs/pull/1114/files/08da994f2d15e01c3d1c9d54c64fe172857dc15d#diff-778ad7ebcca7d8ff16a50b1ef232a622